File: package/main/src/Validate/isString.ts
/**
 * Determines if the value is a string
 * @param  {unknown} value - Value to check
 * @returns boolean - True if the value is a string, false otherwise
 * @example isString("test"); // true
 * isString(123); // false
 */
const isString = (value: unknown): value is string => {
  return typeof value === "string";
};

export { isString };

File: package/umt_python/src/validate/is_string.py
def is_string(value: object) -> bool:
    """
    Determines if the value is a string.

    Args:
        value: Value to check.

    Returns:
        True if the value is a string, False otherwise.

    Example:
        >>> is_string("test")
        True
        >>> is_string(123)
        False
    """
    return isinstance(value, str)
